.github/workflows/issue_event_moves_card.yml

```diff
@@ -0,0 +1,121 @@
+            --header "Authorization: Bearer ${{ secrets.PAT }}" \
+            --data "{\"query\":\"{ node(id: \\\"$project_id\\\") { ... on ProjectV2 { items(first: 100) { nodes { id fieldValues(first: 8) { nodes { ... on ProjectV2ItemFieldTextValue { text field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldDateValue { date field { ... on ProjectV2FieldCommon { name } } } ... on ProjectV2ItemFieldSingleSelectValue { name field { ... on ProjectV2FieldCommon { name } } } } } content { ... on Issue { id } } } } fields(first: 20) { nodes { ... on ProjectV2SingleSelectField { id name options { id name } } } } } } }\"}")
+          echo "issues=$project_fields_and_issues"
+          # Extract the issue ids and PVII ids. The issue_id is the global_id, the PVII id is the project specific id
+          issue_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].content.id')
+          PVII_ids=$(echo $project_fields_and_issues | jq -r '.data.node.items.nodes[].id')
+          # In the following loop we need both values, so we combine them into a single string
+          combined_ids=$(paste -d, <(echo "$issue_ids") <(echo "$PVII_ids"))
+          for combined_id in $combined_ids; do
+            issue_id=$(echo $combined_id | cut -d, -f1)
+            echo "issue_id is $issue_id"
+            PVII_id=$(echo $combined_id | cut -d, -f2)
+            echo "PVII_id is $PVII_id"
+            if [ "$issue_id" == "$ISSUE_NODE_ID" ]; then
+              # Extract the status field id
+              status_field_id=$(echo $project_fields_and_issues | jq -r '.data.node.fields.nodes[] | select(.name == "Status") | .id')
+              # Get the current status of the issue
+              current_status=$(echo $project_fields_and_issues | jq -r --arg PVII_id "$PVII_id" '.data.node.items.nodes[] | select(.id == $PVII_id) | .fieldValues.nodes[] | select(.field.name == "Status") | .name')
+              echo "current_status is $current_status"
+              priority_message=""
+              workload_message=""
+              status_message=""
+              if [ "$current_status" == "$CURRENT_COLUMN_NAME" ]; then
+                # Move the issue to the specified column
+                Option_id=$(echo $project_fields_and_issues | jq -r --arg COLUMN_NAME "$COLUMN_NAME" '.data.node.fields.nodes[] | select(.name == "Status") | .options[] | select(.name == $COLUMN_NAME) | .id')
+                echo "Option_id is $Option_id"
+                echo "change field $status_field_id of issue $issue_id in project $project_id to $COLUMN_NAME"
+                status_message="updateProjectV2ItemFieldValue: updateProjectV2ItemFieldValue(input: {projectId: \"'$project_id'\", itemId: \"'$PVII_id'\", fieldId: \"'$status_field_id'\", value: { singleSelectOptionId: \"'$Option_id'\"}}) {projectV2Item {id}}"
+                echo "status_message is $status_message"
+              fi
+              if [ "$PRIORITY" != 'NONE' ]; then
```

Comment:
If we unlabel an issue, the old labels are still visible in the table of the project. Do you think this is a problem?